pyquil/api/_qpu.py

```diff
@@ -138,35 +175,34 @@ def run(self, run_priority: Optional[int] = None):
         only do measurements where there is a 1-to-1 mapping between qubits and classical
         addresses.
 
-        :param run_priority: The priority with which to insert jobs into the QPU queue. Lower
-                             integers correspond to higher priority. If not specified, the QPU
-                             object's default priority is used.
+        :param run_priority: (Deprecated) Unused, argument provided for backwards compatibility
         :return: The QPU object itself.
         """
         # This prevents a common error where users expect QVM.run()
         # and QPU.run() to be interchangeable. QPU.run() needs the
         # supplied executable to have been compiled, QVM.run() does not.
         if isinstance(self._executable, Program):
-            raise TypeError("It looks like you have provided a Program where an Executable"
-                            " is expected. Please use QuantumComputer.compile() to compile"
-                            " your program.")
+            raise TypeError(
+                "It looks like you have provided a Program where an Executable"
+                " is expected. Please use QuantumComputer.compile() to compile"
+                " your program.")
         super().run()
 
         request = QPURequest(program=self._executable.program,
                              patch_values=self._build_patch_values(),
                              id=str(uuid.uuid4()))
-        job_priority = run_priority if run_priority is not None else self.priority
-        job_id = self.client.call('execute_qpu_request', request=request, user=self.user,
-                                  priority=job_priority)
+
+        job_id = self.client.call('execute_qpu_request', request=request)
```

Comment:
it looks like `self.user` is deprecated now?

Reply:
Yes, how should we handle that?

Reply:
why not keep passing the priority and user to the request? if the service truly doesnt care about it then it will just ignore it